packages/gatsby/src/bootstrap/load-plugins/validate.ts

```diff
@@ -300,11 +308,39 @@ async function validatePluginsOptions(
           })
         }
 
-        plugin.options = await validateOptionsSchema(
+        const { value, warning } = await validateOptionsSchema(
           optionsSchema,
           (plugin.options as IPluginInfoOptions) || {}
         )
 
+        plugin.options = value
+
+        // Handle unknown key warnings
+        const validationWarnings = warning?.details
+
+        if (validationWarnings?.length > 0) {
+          reporter.warn(
+            stripIndent(`
+        Warning: there are unknown plugin options for "${plugin.resolve}"${
+              configDir ? `, configured by ${configDir}` : ``
+            }: ${validationWarnings
+              .map(error => error.path.join(`.`))
+              .join(`, `)}
+        Please open an issue at ghub.io/${
```

Comment:
Hesitant about using `ghub.io` here since this would also log for local plugins and redirect to a 404 on npm, but it is useful for handling official and community plugins. If we do keep it it would be good to prepend `https://` so smarter terminals highlight it as a clickable url.

Reply:
This is [existing code](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/bootstrap/load-plugins/validate.ts#L362-L364). I'll add the HTTPS but removing/changing it seems out of scope.

Reply:
Thanks for adding that!